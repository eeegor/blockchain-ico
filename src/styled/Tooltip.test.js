import React from 'react';
import renderer from 'react-test-renderer';
import 'jest-styled-components';
import { Tooltip } from '.';

describe('Tooltip', () => {
	test('renders default tooltip', () => {
		const tree = renderer.create(<Tooltip>Title</Tooltip>).toJSON();
		expect(tree).toMatchSnapshot();
	});

	test('renders tooltip for mobile', () => {
		const tree = renderer
			.create(<Tooltip color="green">Title</Tooltip>)
			.toJSON();
		expect(tree).toMatchSnapshot();
		expect(tree).toHaveStyleRule('top', '12px');
	});
});
