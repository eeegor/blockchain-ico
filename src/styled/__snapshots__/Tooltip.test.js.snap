// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tooltip renders default tooltip 1`] = `
.c0 {
  position: fixed;
  top: 12px;
  left: 12px;
  right: 12px;
  color: #fff;
  z-index: 12000;
  background: #1A2440;
  display: grid;
  grid-template-columns: 50px auto 100px;
  grid-gap: 0;
}

.c0 .c1 {
  margin-right: 12px;
  margin-bottom: 6px;
}

.c0 .c3 {
  margin-bottom: 0;
  margin-right: 6px;
  color: #fff;
  font-size: 75%;
  opacity: 0.8;
}

.c0 .c2 {
  margin-bottom: 0;
  color: #fff;
  font-size: 75%;
  opacity: 0.5;
}

@media (min-width:767px) {
  .c0 {
    padding: 12px;
    grid-template-columns: 60px 180px 140px auto 120px;
  }
}

<div
  className="c0"
>
  Title
</div>
`;

exports[`Tooltip renders tooltip for mobile 1`] = `
.c0 {
  position: fixed;
  top: 12px;
  left: 12px;
  right: 12px;
  color: #fff;
  z-index: 12000;
  background: #1A2440;
  display: grid;
  grid-template-columns: 50px auto 100px;
  grid-gap: 0;
}

.c0 .c1 {
  margin-right: 12px;
  margin-bottom: 6px;
}

.c0 .c3 {
  margin-bottom: 0;
  margin-right: 6px;
  color: #fff;
  font-size: 75%;
  opacity: 0.8;
}

.c0 .c2 {
  margin-bottom: 0;
  color: #fff;
  font-size: 75%;
  opacity: 0.5;
}

@media (min-width:767px) {
  .c0 {
    padding: 12px;
    grid-template-columns: 60px 180px 140px auto 120px;
  }
}

<div
  className="c0"
  color="green"
>
  Title
</div>
`;
